app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -181,6 +181,7 @@ def show
     param :id, :identifier, :required => true, :desc => N_("repository ID")
     param :source_url, String, :desc => N_("temporarily override feed URL for sync"), :required => false
     param :incremental, :bool, :desc => N_("perform an incremental import"), :required => false
+    param :force_full, :bool, :desc => N_("Force sync even if no upstream changes are detected"), :required => false
```

Comment:
This comes from the pulp option (https://docs.pulpproject.org/en/2.13/nightly/user-guide/release-notes/2.10.x.html).  I was attempting to keep these the same for simplicity, but if you really dislike it, we can change it.